package.json

```diff
@@ -22,5 +22,11 @@
     "shell"
   ],
   "author": "Keith Cirkel <npm@keithcirkel.co.uk> (http://keithcirkel.co.uk/)",
-  "license": "MIT"
+  "license": "MIT",
+  "scripts":{
+    "test1": "node ./index.js \"node test/willNotExit.js\"",
+    "test2": "node ./index.js --verbose \"node test/willThrowError.js\" \"node test/willNotExit.js\"",
+    "test3": "node ./index.js --verbose --wait \"node test/willThrowError.js\" \"node test/willNotExit.js\"",
+    "test4": "node ./index.js --help"
```

Comment:
If you could add `"test": "for i in {1..4}; do npm run test$i; done"` as a script - and then ensure all of the tests are passable, that'd be great :smile: